docs/sources/datasources/prometheus.md

```diff
@@ -27,6 +27,23 @@ To access Prometheus settings, hover your mouse over the **Configuration** (gear
 | `Disable metrics lookup`  | Checking this option will disable the metrics chooser and metric/label support in the query field's autocomplete. This helps if you have performance issues with bigger Prometheus instances.           |
 | `Custom Query Parameters` | Add custom parameters to the Prometheus query URL. For example `timeout`, `partial_response`, `dedup`, or `max_source_resolution`. Multiple parameters should be concatenated together with an '&amp;'. |
 
+### Exemplars
+
+> **Note:** This feature is available in Prometheus 2.25+.
+
+The Exemplars configuration allows you to add multiple endpoints to any of your Exemplar label.
+
+You can use this functionality to link to your tracing backend from an Exemplar.
+{{< docs-imagebox img="/img/docs/v74/exemplars-settings.png" class="docs-image--no-shadow" caption="Screenshot of the Exemplars configuration" >}}
```

Comment:
It would be great if we can add a link to tracing doc content from here.

Reply:
Which one?

Reply:
You can list the data sources and explore where the "trace to logs" feature is available. 

https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-logs
https://grafana.com/docs/grafana/latest/explore/#trace-to-logs
https://grafana.com/docs/grafana/latest/datasources/zipkin/#trace-to-logs
https://grafana.com/docs/grafana/latest/datasources/jaeger/#trace-to-logs

Reply:
I'm not sure how the trace-to-logs feature is related to this sentence. Can you help me rephrase it?